content/en/metrics/distributions.md

```diff
@@ -38,27 +38,27 @@ See the [Developer Tools section][1] for more implementation details.
 
 **Note:** Because distributions are a new metric type, they should be instrumented under new metric names during submission to Datadog.
 
-## Enabling percentile aggregations
+## Enabling advanced query functionality
 
 Like other metric types, such as `gauges` or `histograms`, distributions have the following aggregations available: `count`, `min`, `max`, `sum`, and `avg`. Distributions are initially tagged the same way as other metrics, with custom tags set in code. They are then resolved to host tags based on the host that reported the metric. 
 
-However, you can additionally enable the calculation of globally accurate percentile aggregations for all queryable tags on your distribution on the Metrics Summary page. This provides aggregations for `p50`, `p75`, `p90`, `p95`, and `p99` as well as advanced query options like threshold queries for SLOs.
+However, you can  enable advanced query functionality such as the calculation of globally accurate percentile aggregations for all queryable tags on your distribution on the Metrics Summary page. This provides aggregations for `p50`, `p75`, `p90`, `p95`, and `p99` or any user-defined percentile of your choosing (with up to two decimal points such as 99.99). Enabling advanced queries also unlocks threshold queries.
 
-{{< img src="metrics/distributions/percentiles.mp4" alt="Enable Percentiles" video=true style="width:80%;">}}
+{{< img src="metrics/distributions/advancedquery.mp4" alt="Advanced Query Functionality" video=true style="width:80%;">}}
```

Comment:
```suggestion
{{< img src="metrics/distributions/advancedquery.mp4" alt="A user enabling advanced query functionality by clicking on edit under the advanced section" video=true >}}
```

Reply:
On the page, this video looks zoomed in and the quality isn't great. Could you try recording it when zoomed out a bit?